Docs and link updates for plan, connect, and usage docs (#18344)
cockroachdb/docs: src/current/cockroachcloud/troubleshooting-page.md

This page describes common CockroachDB {{ site.data.products.cloud }} errors and their solutions.

{{site.data.alerts.callout_danger}}

> We have updated the CA certificate used by CockroachDB {{ site.data.products.basic }} and {{ site.data.products.standard }} clusters. If you downloaded this certificate prior to June 17, 2021, **you must [download the updated certificate]({% link cockroachcloud/connect-to-your-cluster.md %}#connect-to-your-cluster) by September 30, 2021** to avoid disruptions to your service.

Reviewer: ```suggestion
On June 17, 2021, Cockroach Labs updated the CA certificate used by CockroachDB {{ site.data.products.basic }} and {{ site.data.products.standard }} clusters. If you downloaded this certificate prior to June 17, 2021, **you must [download the updated certificate]({% link cockroachcloud/connect-to-your-cluster.md %}#connect-to-your-cluster) by September 30, 2021** to avoid disruptions to your service.
```
